### PR TITLE
Complete the Random module

### PR DESCRIPTION
**Pull request by dmonzonis**
Add more RNG related methods.

### CODE REVIEW

File: src/lazarus/Random.h [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+
+    /**
+     * Return true with a 1 in n probability.
+     */
+    static bool oneIn(unsigned n);
+
+    /**
+     * Return a random number generated from a normal distribution.
+     * 
+     * The normal (or Gaussian) distribution will use the given mean and
```
Comment (Elkasitu) [inline src/lazarus/Random.h:100]: nerd

File: src/lazarus/Random.h [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+};
+
+template <typename C, typename T = typename C::value_type>
+T& Random::choice(C& container)
+{
+    // Use size() instead of empty() to make conditions less restrictive
+    size_t size = container.size();
+    if (size == 0)
+        // TODO: Use own exception class
+        throw new std::runtime_error("Container is empty");
```
Comment (Elkasitu) [inline src/lazarus/Random.h]: Maybe call it `RandomException` or if you wanna be funny `LuckException`

**Review by Elkasitu — APPROVED** [review]
OK but idk how to use `Random::normal` or what its purpose is. but that's just me being a brainlet

**Comment by dmonzonis** [comment]
The normal distribution can be used, for example, for bullet dispersion.

**Pull request merged**